include/quic/endpoint.hpp

```diff
@@ -36,14 +36,19 @@ namespace oxen::quic
         friend class Connection;
         friend class Stream;
 
-      private:
```

Comment:
private *is* default for classes, but I don't see an issue with being explicit about it.  That said, I don't have strong feelings either way.